### ISSUE TITLE
Fake surface not getting removed. Not able to do anything on phone after picture is clicked

### ISSUE THREAD

**Issue by Kunal1093** [kevalpatel2106/android-hidden-camera#36]
In O OS, after the picture is clicked in service, I am not able to access the device until I restart it. I think the fake surface is not getting removed as I am not able to swipe or open any other app, but I am able to access notification panel and also notification is showing in notification panel that my app is displaying over other apps continuously. Please help

**Comment by Rloguzzo** [comment]
up

**Comment by in-skyadav** [comment]
I am getting the same error, once I took the picture touch on whole window view is disabled and I am not able to do anything...
only notification panel and bottom navigation bar touch is working.

**Comment by androidnoob123** [comment]
Hi guys, I had similar problems with this library. I used the one linked below instead and it worked fine for me in Android 0. See the closed issue over there where I have posted how to make it take multiple pictures.

https://github.com/KAditi/Camera-Application

Regarding the window view disabled problem, I think the camera open (or maybe something else?) is working on the main thread and its blocking the UI. The source code for the above app is similar (although AsyncTask is used) so you may be able to solve it by looking into its source code. Hope it works for you!

**Comment by al-shivani** [comment]
I am facing the same issue, but the sample app is working fine.
@Kunal1093 did you get any other solution for this.

**Comment by Pognio10** [comment]
does anyone have a solution?

**Comment by tjhannes** [comment]
I had the same issue but using the latest version from this repo and including it as a project helps
`implementation project(':hiddencamera')`

**Comment by kalakoff** [comment]
any solution for this issue?

**Comment by tjhannes** [comment]
Yes, just use the most recent files from the github repo (instead of version 1.3.3 as mentioned in the readme instructions). Now everything works fine for me

**Comment by al-shivani** [comment]
Hi tjhannes 

I did the same, Its working good for all devices except OnePlus 5. Do you have any solution for same?

**Comment by tjhannes** [comment]
I tested it only on Huawei P20, not sure about other devices
